### ISSUE TITLE
Add Libre Translate instead of Google Translate for page translation

### ISSUE THREAD

**Issue by dmikushin** [Alex313031/thorium#846]
I'm always frustrated when Google Translate is collecting my data, when I translate the content of web pages in the browser.

I want to Thorium Browser to provide an option to use self-hosted (LibreTranslate)[https://github.com/LibreTranslate/LibreTranslate] as a backend for pages translation in Thorium.

**Comment by dmikushin** [comment]
> Thank you for your suggestion, but considering factors including practical and financial factors, we will continue to use Google Translate.

Financial factors... what? Who are you, btw?

**Comment by drag0n1zed** [comment]
I think they meant to add an option to use libretranslate on their own self-hosted platforms, not by the collaborators on this repo

**Comment by dmikushin** [comment]
Yes, I only asked for the addition of a setting to select the translation service.

However, the discussion from the very beginning took a direction not typical for open projects. I can't recall a single hobby project where money was mentioned in the first response to a feature request. And for the response to come from a 'partner' of the developer, whose contribution to the project is hard to trace. The developer's busyness, as well as reduced interest or communication skills, are quite common. But from collaborators who assist him at his request, it would be logical to expect more rational behavior. I mean, a rational response to a request to a hobby project is to discuss technical details. I was genuinely surprised that this did not follow.

But perhaps we can help the project's collaborators find answers to some obvious questions. About the fact that any funding for such a project will be scrutinized for user data trading and espionage. One must be very careless (or very inexperienced) collaborator to even hint at money in connection with a product that touches private data. Such discussions made me feel vulnerable from the fact that, as it turns out, the thorium project is ready to do anything for money. I would like this ticket to serve as a cautionary example for all collaborators on how to damage the reputation of a project with a single short comment.
